src/components/Frame/Frame.tsx

```diff
@@ -351,10 +351,6 @@ class Frame extends React.PureComponent<CombinedProps, State> {
     }
   };
 
-  private handleClick = () => {
```

Comment:
In this case, we do want to preserve the focus handling, since browsers handle this a bit differently for on-page links. Safari in particular doesn't actually move focus by itself.

[You can see this happening most easily with VoiceOver running (video).](https://drive.google.com/open?id=1BwV3roTf3RGRUwcgwCjc3tLE208HRyKH) In this video I'm tabbing to the "Skip to content" link, pressing enter to fire it, and then while keyboard focus does technically skip to the `<main>` container, VoiceOver's cursor moves to the next element in the DOM after the link is hidden again. The next keystroke is shift + tabbing backwards, to show how the keyboard focus order makes sense if focus was in the main section, but the VoiceOver cursor appears to jump down to the end of the nav sidebar.

Safari on iOS behaves the same way.

Let me know if that makes sense! I'm happy to spend some office hours time next week testing this together.

Reply:
I've added focus on the main element. However in Safari Voiceover tells you to click `ctrl-option-spacebar` to follow the link. Doing that works great, pushing `enter` alone does not. Is that as expected?

Reply:
> Doing that works great, pushing enter alone does not. Is that as expected?

I'm able to activate the link with Enter, as well as use the VO-specific recommended keys. However, either causes a page refresh instead of moving focus.

I'm curious as to why this worked as a button but not a link. Do you have some time before Thursday to test through this together during office hours? @dleroux

Reply:
adding `tabIndex={-1}` did the trick.

As for jumping out of the frame in the playground, I have a feeling that the playground is doing that because the behavior was the same before this change.

Building to web shows that this is working well there.